modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -324,9 +346,8 @@ def min(self, **kwargs):
     def sum(self, **kwargs):
         min_count = kwargs.pop("min_count")
         if min_count != 0:
-            return super().sum(
-                min_count=min_count,
-                **kwargs,
+            raise NotImplementedError(
+                f"OmniSci' sum does not support such set of parameters: min_count={min_count}"
```

Comment:
```suggestion
                f"OmniSci's sum does not support such set of parameters: min_count={min_count}"
```
here and in other places

Reply:
dot at the end :)